lib/talon/page_controller.ex

```diff
@@ -5,18 +5,30 @@ defmodule Talon.PageController do
       talon = opts[:concern] || raise("concern option required")
       plug :set_concern, talon: talon
 
-      @spec page(Plug.Conn.t, Map.t) :: Plug.Conn.t
-      def page(conn, params) do
-        page = Map.get(params, "page", "dashboard")
-        render(conn, "#{page}.html")
+
+      @spec index(Plug.Conn.t, Map.t) :: Plug.Conn.t
+      def index(conn, params) do
```

Comment:
I'm not sure why there are two actions here? Don't we need just one? Either index or show.